<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTeacher</class>
 <widget class="QMainWindow" name="EditTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Öğretmen güncelleme sayfası</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 163, 127);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string> Ad :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string> Soyad :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string> Doğum  tarihi :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string>Cinsiyet :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teacher_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teacher_surname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QDateTimeEdit" name="teacher_birthdate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 215, 187);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="kaydet_for_edit_btn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>470</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Güncelle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string> Öğretmen güncelleme sayfası</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bilgileri_goster_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>422</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(17, 57, 70);</string>
     </property>
     <property name="text">
      <string>Bilgileri goster</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="kadin">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>350</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kadın</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="erkek">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Erkek</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 215, 187);</string>
     </property>
     <property name="text">
      <string> Numara:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teacher_number">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 215, 187);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
